tests/components/reolink/test_init.py

```diff
@@ -373,7 +373,8 @@ def mock_supported(ch, capability):
     assert (
         entity_registry.async_get_entity_id(domain, const.DOMAIN, original_id) is None
     )
-    assert entity_registry.async_get_entity_id(domain, const.DOMAIN, new_id)
+    if new_id is not None:
```

Comment:
Can we avoid adding more if statements to the tests?

Reply:
yea, sorry missed this.
Removed it now.